### PR TITLE
Fixed issue with text wrapping WiFiMenuItemView.

### PR DESCRIPTION
**Pull request by ErrorErrorError**
- Created custom icons when displaying "Looking for networks" on StatusMenu
- Created custom warning and error wifi icon

Ready for review!

### CODE REVIEW

**Comment by Goshin** [comment]
Thanks for your great work. May I ask where are these icons from?

@williambj1 do you think we need a dedicated animation for connecting? The current one in master looks good to me, though not identical to that of CoreWLAN.

**Comment by williambj1** [comment]
I can't reach my laptop for the next few days so I really couldn't test the  changes. The code looks good.

**Comment by ErrorErrorError** [comment]
> Thanks for your great work. May I ask where are these icons from?
> 
> 
> 
> @williambj1 do you think we need a dedicated animation for connecting? The current one in master looks good to me, though not identical to that of CoreWLAN.

The WiFi Scanning icons i created them by using WiFiStateOff as the base and the using the respective WiFiSignal and removing the extra padding. Those icons I got t from the repo. 

The WiFiStateWarning and WiFiStateError I created them from scratch so the coloring might be a bit off.

We don’t have to add that animation with new icons, it’s just something I noticed

File: HeliPort/Appearance/WiFiMenuItemView.swift [hunk 1]
```diff
@@ -85,47 +82,48 @@ class WifiMenuItemView: NSView {
...
+        ssidLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor).isActive = true
+
+        lockImage.centerYAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
+        lockImage.leadingAnchor.constraint(equalTo: ssidLabel.trailingAnchor, constant: 10).isActive = true
+        lockImage.widthAnchor.constraint(equalToConstant: 10).isActive = true
+
+        signalImage.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: 1).isActive = true
+        signalImage.leadingAnchor.constraint(equalTo: lockImage.trailingAnchor, constant: 12).isActive = true
+        signalImage.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -12).isActive = true
+        signalImage.widthAnchor.constraint(equalToConstant: 21).isActive = true
```
Comment (Goshin) [inline HeliPort/Appearance/WiFiMenuItemView.swift:115]: Can you adjust the width here to 18? It will make the icon fit in the view better.

**Comment by Goshin** [comment]
Thanks a lot.

**Pull request merged**